app/components/global/AppBar/AppBar.js

```diff
@@ -71,4 +70,18 @@ const AppBar = ({ user, menuItems }) => (
   </AppBarContainer>
 )
 
+AppBar.propTypes = {
+  menuItems: PropTypes.arrayOf(
+    PropTypes.shape({
+      label: PropTypes.string.isRequired,
+      link: PropTypes.string.isRequired,
+    }),
+  ).isRequired,
+  user: PropTypes.shape({}),
+}
+
+AppBar.defaultProps = {
+  user: null,
```

Comment:
What would you think of `user = {}` rather than `user = null`?

Reply:
The `ProfileMenu` component checks if user exists before displaying either a sign in link or the profile menu. Defaulting it to an object would require these checks to be changed to something like `Object.keys(user).length` which if quite a bit more verbose. Would there be any benefit of casting an empty object to an undefined prop?